### ISSUE TITLE
Error when passing long array to model

### ISSUE THREAD

**Issue by chameroy** [TuringLang/Turing.jl#418]
Hi, I'm trying to run my first model using the following code:

x = Array(d2[:height]) . # this produces an array of length 352 which is then passed to the model below

@model howell1(height) = begin
    μ ~ Normal(178, 20)
    σ ~ Uniform(0, 50)
    [height[i] ~ Normal(μ, σ) for i = 1:length(height)]
    return μ, σ
end

I've tried sampling using the algorithms and parameters used in the introduction and I frequently get the following warning/error:

[Turing.WARNING]: Numerical error has been found in gradients.
 verifygrad(::Array{Float64,1}) at ad.jl:87

Am I doing something wrong here?

**Comment by xukai92** [comment]
I don't know if list comprehension will cause a problem. Maybe try

```julia
@model howell1(height) = begin
μ ~ Normal(178, 20)
σ ~ Uniform(0, 50)
for i = 1:length(height)
    height[i] ~ Normal(μ, σ)
end
return μ, σ
end
```

Also what is the sampling algorithm you use? It also might be too large stepsize you set.

**Comment by yebai** [comment]
This issue should be fixed by v0.7. Please re-open this issue if the same issue happens again.
